web/src/compose_call_ui.ts

```diff
@@ -10,38 +11,53 @@ import {current_user, realm} from "./state_data";
 import * as ui_report from "./ui_report";
 import * as util from "./util";
 
-export function update_audio_and_video_chat_button_display() {
+const call_response_schema = z.object({
+    msg: z.string(),
+    result: z.string(),
+    url: z.string(),
+});
+
+export function update_audio_and_video_chat_button_display(): void {
     update_audio_chat_button_display();
     update_video_chat_button_display();
 }
 
-export function update_video_chat_button_display() {
+export function update_video_chat_button_display(): void {
     const show_video_chat_button = compose_call.compute_show_video_chat_button();
     $(".compose-control-buttons-container .video_link").toggle(show_video_chat_button);
     $(".message-edit-feature-group .video_link").toggle(show_video_chat_button);
 }
 
-export function update_audio_chat_button_display() {
+export function update_audio_chat_button_display(): void {
     const show_audio_chat_button = compose_call.compute_show_audio_chat_button();
     $(".compose-control-buttons-container .audio_link").toggle(show_audio_chat_button);
     $(".message-edit-feature-group .audio_link").toggle(show_audio_chat_button);
 }
 
-function insert_video_call_url(url, target_textarea) {
+function insert_video_call_url(
+    url: string,
+    target_textarea: JQuery<HTMLTextAreaElement> | undefined,
+): void {
     const link_text = $t({defaultMessage: "Join video call."});
     compose_ui.insert_syntax_and_focus(`[${link_text}](${url})`, target_textarea, "block", 1);
 }
 
-function insert_audio_call_url(url, target_textarea) {
+function insert_audio_call_url(
+    url: string,
+    target_textarea: JQuery<HTMLTextAreaElement> | undefined,
```

Comment:
`target_textarea` => `$target_textarea` should be done in a follow-up commit, since we have a convention of naming variables that way.